packages/wallet-sdk/src/sign/walletlink/relay/type/WalletLinkSession.ts

```diff
@@ -21,10 +21,9 @@ export class WalletLinkSession {
     this._id = id || randomBytesHex(16);
     this._secret = secret || randomBytesHex(32);
 
-    this._key = sha256
-      .create()
-      .update(`${this._id}, ${this._secret} WalletLink`) // ensure old sessions stay connected
-      .hex();
+    const hash = sha256.create().update(`${this._id}, ${this._secret} WalletLink`).digest();
+
+    this._key = Array.from(hash, (byte) => byte.toString(16).padStart(2, '0')).join('');
```

Comment:
`import { bytesToHex } from '@noble/hashes/utils'` is faster

Reply:
thanks for the suggestion, will use that! didn't know noble hashes already include this util

Reply:
fixed